src/index.html

```diff
@@ -13,6 +13,39 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <h3 class="card__product-id">Product code: 195434</h3>
+
+      <div class="card__reviews">
+        <div class="card__stars">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <div class="price">
```

Comment:
The 'price' block should be an element of 'card', so it should follow the BEM naming convention. For example, 'card__price' instead of just 'price'.